interface.go

```diff
@@ -129,6 +136,13 @@ type JobTimeInfo struct {
 	MaxTime    time.Duration `bson:"max_time" json:"max_time,omitempty" yaml:"max_time,omitempty"`
 }
 
+// JobRetryInfo stores configuration and information for a job that can retry.
+// Support for retrying jobs is optional for Queue implementations.
+type JobRetryInfo struct {
```

Comment:
Why do we need both `Retryable` and `RetryCount`? I think having a single configuration called `RetryCount` or `Retries` would be simpler.

Reply:
`Retryable` tells you whether the job can retry at all. `RetryCount` is not meant to be modified by you - it tells you how many times it's tried in total so far.

Reply:
Ohh I thought it was like the max retry attempts allowed. Can we set a max?

Reply:
[EVG-13544](https://jira.mongodb.org/browse/EVG-13544)